### PR TITLE
Add anyOf and oneOf

### PR DESCRIPTION
**Pull request by Alan-Cha**

### CODE REVIEW

**Comment by Alan-Cha** [comment]
WIP

File: packages/openapi-to-graphql/src/schema_builder.ts [hunk 1]
```diff
@@ -266,6 +278,14 @@ function createOrReuseOt({
...
         })
       }
+      // },
+      // isTypeOf: (source, context, info) => {
+      //   console.log('source: ', JSON.stringify(source, null, 2))
+
+      //   console.log(Object.keys(source))
+
+      //   return false
+      // }
```
Comment (ErikWittern) [inline packages/openapi-to-graphql/src/schema_builder.ts]: Remove?

File: packages/openapi-to-graphql/src/schema_builder.ts [hunk 2]
```diff
@@ -303,7 +323,163 @@ function createOrReuseOt({
...
+    }
+  })
+
+  for (let i = 0; i < types.length - 1; i++) {
+    const currentType = types[i]
+
+    for (let j = i + 1; j < types.length; j++) {
+      const otherType = types[j]
+
+      console.log('testing')
```
Comment (ErikWittern) [inline packages/openapi-to-graphql/src/schema_builder.ts]: No more testing!

File: packages/openapi-to-graphql/src/types/operation.ts [hunk 3]
```diff
@@ -38,18 +42,26 @@ export type DataDefinition = {
    *
    * Or if the dataDef is an object type, the subDefinitions are references to
    * the field types
+   *
```
Comment (Alan-Cha) [inline packages/openapi-to-graphql/src/types/operation.ts]: Extra comment

Comment (Alan-Cha) [reply packages/openapi-to-graphql/src/types/operation.ts]: Oh, sorry, this is intentional.

**Comment by Alan-Cha** [comment]
Closed in favor of #287

**Pull request closed without merging**